_maps/map_files/Delta/delta.dmm

```diff
@@ -69674,7 +69676,7 @@
 	dir = 9
 	},
 /obj/machinery/hologram/holopad{
-	pixel_y = -16
+	pixel_y = 0
```

Comment:
please correctly remove these null entries as requested in mapping standards. We do not want 0 px variables in our map files.

Reply:
(Note, this happens due to using DM to mapedit, as opposed to SDMM)